scripts/run_e2e_tests.sh

```diff
@@ -173,7 +173,7 @@ node_modules/.bin/webdriver-manager start --versions.chrome 2.41 --detach --quie
 # TODO(jacob): Find a webdriver or selenium argument that controls log level.
 (node_modules/.bin/webdriver-manager start 2>/dev/null)&
 # Start a demo server.
-(python ../oppia_tools/google_appengine_1.9.67/google_appengine/dev_appserver.py --host=0.0.0.0 --port=9001 --clear_datastore=yes --dev_appserver_log_level=critical --log_level=critical --skip_sdk_update_check=true $APP_YAML_FILEPATH)&
+(python ../oppia_tools/google_appengine_1.9.67/google_appengine/dev_appserver.py --host=0.0.0.0 --port=9001 --clear_datastore=yes --dev_appserver_log_level=critical --log_level=critical --skip_sdk_update_check=true $APP_YAML_FILEPATH > log.txt)&
```

Comment:
/cc @cuichenli FYI, please note changes to e2e test script in this file.

Reply:
My concern with this one is that this might swallow up error logs on CI too.

Reply:
No, actually it doesn't. The CI logs don't print error log from the server at all. The intention was to capture server logs but even that isn't necessary. If we keep the log level to info or debug then it prints server log in CI log itself, which is sufficient for debugging. I will remove this entire file change.

Reply:
Done.

Reply:
Ah ok, sg. @cuichenli please ignore my comment, sorry!